tests/python/nightly/relax/test_frontend_from_fx.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import torch
+from torch import fx
+from torch.nn import Module
+
+import tvm
+from tvm import relax
+import tvm.testing
+from tvm.relax.frontend.torch import from_fx
```

Comment:
I think we might not need this nightly test, as the test case changes in tests/python/relax seem sufficient to cover this.

Reply:
Ok. Removed.